templates/media/media-gallery-item.php

```diff
@@ -33,7 +33,10 @@ class="<?php echo esc_attr( apply_filters( 'rtmedia_gallery_list_item_a_class',
 			global $rtmedia_query;
 
 			$alt_text      = rtmedia_image_alt( false, false );
-			$rtmedia_media = $rtmedia_query->rtmedia;
+			$rtmedia_media = '';
+                        if ( ! empty( $rtmedia_query ) && isset( $rtmedia_query->rtmedia ) ) {
```

Comment:
Used ternary operator to wrap this in one line.

Reply:
This changes approved by @chandrapatel . LINK: https://github.com/rtMediaWP/rtMedia/pull/1055 @milindmore22

Reply:
why are those changes in this PR?